src/EA.Iws.Core/Authorization/Permissions/ExportNotificationPermissions.cs

```diff
@@ -36,8 +36,11 @@ public static class ExportNotificationPermissions
 
         public const string CanEditInternalComments = "Can edit internal comments";
 
+        public const string CanEditEwcCodes = "Can edit EWC codes";
         public const string CanEditYCodes = "Can Edit Y Codes";
 
         public const string CanEditOperationCodes = "Can edit operation codes";
+        
+        
```

Comment:
Need to remove multiple blank lines in a row, also especially that this is before a closing curly bracket.